Removed many conditionals required before Chrome 89 from the COOP/COEP article (#4813)
GoogleChrome/web.dev: src/site/content/en/blog/coop-coep/index.md

**Updates**

> - **March 5, 2021**: Removed limitations for SharedArrayBuffer, debugging
>   functionalities, etc. They are now fully enabled in Chrome 89. Added
>   `performance.now()` and `performance.timeOrigin` as the upcoming capability
>   that will have better precision.

Reviewer: ```suggestion
- **March 5, 2021**: Removed limitations for `SharedArrayBuffer`, `performance.measureUserAgentSpecificMemory()`, and debugging
  functionalities, which are now fully enabled in Chrome 89. Added
  upcoming capabilities, `performance.now()` and `performance.timeOrigin`,
  that will have higher precision.
```

**Updates**

> - **March 5, 2021**: Removed many conditionals required before Chrome 89.

Reviewer: I suggest being explicit in what's removed from the post. Something like: "Removed limitations for SharedArrayBuffer, debugging functionalities, etc. that are now fully enabled in Chrome 89."
Since there's no archived version of the post, this update is our chance to preserve the history of changes.

Author: Makes sense. I will.

Author: Fixed. Please check.

Reviewer: I wanted to avoid the "etc" and really list all of the removals in this update. I added another suggestion below.

> - **February 19, 2021**: Added a note about feature policy
>   `allow="cross-origin-isolated"` and debugging functionality on DevTools.

Reviewer: ```suggestion
```

Reviewer: Since this info is now removed from the post, I'd remove the update note too.

cross-origin isolated. With a cross-origin isolated state, the webpage will be
able to use privileged features including:

<div class="w-table-wrapper">
  <table>
    <thead>
      <tr>

>         <th>API</th>

Reviewer 2: We might add performance.now() and performance.timeOrigin here as well?

https://bugs.chromium.org/p/chromium/issues/detail?id=1180178

Author: I did once, but removed because I wasn't too sure about these. I'll do.

Author: Fixed. Please have a look.
https://deploy-preview-4813--web-dev-staging.netlify.app/coop-coep/

cross-origin isolated. With a cross-origin isolated state, the webpage will be
able to use privileged features including:

<div class="w-table-wrapper">
  <table>
    <thead>
      <tr>
        <th>API</th>
        <th>Description</th>
      </tr>
    </thead>
    <tbody>
      <tr>
        <td>
          <a href="https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/SharedArrayBuffer">
          <code>SharedArrayBuffer</code></a>
        </td>
        <td>
          Required for WebAssembly Threads. This is available from Android
          Chrome 88. Desktop version is currently enabled by default with the
          help of <a href="https://www.chromium.org/Home/chromium-security/site-isolation">
          Site Isolation</a>, but will require the cross-origin isolated state
          and will be disabled by default in Chrome 91.
        </td>
      </tr>
      <tr>
        <td>
          <a href="/monitor-total-page-memory-usage/">
          <code>performance.measureUserAgentSpecificMemory()</code></a>
        </td>
        <td>

>           Available in from Chrome 89.

Reviewer: ```suggestion
          Available from Chrome 89.
```

        </td>
      </tr>
      <tr>
        <td>
          <a href="https://wicg.github.io/js-self-profiling/">
          JS Self-Profiling API</a>
        </td>
        <td>
          Not available in any browsers yet.
        </td>
      </tr>
      <tr>
        <td>
          <a href="https://crbug.com/1180178">
          <code>performance.now()</code>, <code>performance.timeOrigin</code></a>
        </td>
        <td>

>           Both are available on many browsers, but the resolution is limited to
>           100 microseconds or higher. With cross-origin isolation, the
>           resolution can be 5 microseconds or higher.

Reviewer: ```suggestion
          Currently available in many browsers with the resolution limited to
          100 microseconds or higher. With cross-origin isolation, the
          resolution can be 5 microseconds or higher.
```
